src/query/scan_manager.h

```diff
@@ -304,18 +304,18 @@ typedef struct scan_stats SCAN_STATS;
 struct scan_stats
 {
   struct timeval elapsed_scan;
-  UINT64 num_fetches;
-  UINT64 num_ioreads;
+  INT64 num_fetches;
```

Comment:
if this values never be minus, how about remain data type as UINT64 ?
If you agree with me, I think that you are better to change the following changed data type to UINT64.

Reply:
I fixed their data types as UINT64 and printing them by `%llu`.